core/io/image.cpp

```diff
@@ -2697,24 +2697,55 @@ void Image::blend_rect_mask(const Ref<Image> &p_src, const Ref<Image> &p_mask, c
 	}
 }
 
-void Image::fill(const Color &c) {
+// Makes pixel being repeated p_count times in consecutive memory.
+// Results in original pixel and (p_count - 1) subsequent copies of it.
+void Image::_repeat_pixel_over_subsequent_memory(uint8_t *p_pixel, int p_pixelsize, int p_count) {
```

Comment:
```suggestion
// Repeats `p_pixel` `p_count` times in consecutive memory.
// Results in the original pixel and `p_count - 1` subsequent copies of it.
void Image::_repeat_pixel_over_subsequent_memory(uint8_t *p_pixel, int p_pixel_size, int p_count) {
```
One more knit pick 😄

(the official comment style guide suggests using backticks for variables/values https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#comment-style-guide)

Reply:
That `p_pixelsize` is inspired by:
https://github.com/godotengine/godot/blob/4292de21601d31923d0cbf28b1e9813fcaa0a57d/core/io/image.h#L193-L194

I guess I should change it too. Oh, and they're not using `memcpy()` too:
https://github.com/godotengine/godot/blob/4292de21601d31923d0cbf28b1e9813fcaa0a57d/core/io/image.cpp#L89-L103
I guess... another small change? :smile:

Reply:
If it's not that much work I'd just go ahead and change those too :) The whole `Image` class could probably use a bit of review, but that's out of scope for this PR.